pages/common/sops.md

```diff
@@ -0,0 +1,29 @@
+# sops
+
+> SOPS: Secrets OPerationS.
+> Simple and flexible tool for managing secrets.
+> More information: <https://github.com/mozilla/sops>.
+
+- Encrypt a file:
+
+`sops -e {{path/to/myfile.json}} > {{path/to/myfile.enc.json}}`
+
+- To decrypt a file in a `cat` fashion, use the `-d` flag:
```

Comment:
Do not use infinitive here

Reply:
might be worth including some extra information here, @zdroid. For example, we could include a suggested change?

Reply:
@sbrl  Can you elaborate on what is the extra information?